OpenRA.Game/Settings.cs

```diff
@@ -336,9 +336,16 @@ public class ChatSettings
 		public bool ConnectAutomatically = false;
 	}
 
+	public class LobbySettings
+	{
+		public string Faction = "Random";
```

Comment:
Yes, I agree that this is an important part of the parallel installation rework, and i'm planning to put some proper thought into this and file a PR dedicated to that after #13223 and #13226 are merged.

We can then rebase this one on top, so that we don't mix fundamental infrastructure changes with new mod-level features.